src/python/phenix_apps/apps/scorch/kafka/kafka.py

```diff
@@ -0,0 +1,167 @@
+import json, itertools, threading, time, sys, csv, re, os
+
+from phenix_apps.apps.scorch import ComponentBase
+from phenix_apps.common import logger, utils
+from kafka import KafkaConsumer
+from kafka.errors import KafkaError
+from datetime import datetime
+from queue import Queue
+
+scorch_kafka_running = False
+
+class kafka(ComponentBase):
+    def __init__(self):
+        ComponentBase.__init__(self, 'kafka')
+        self.execute_stage()
+
+    def helper(self, csvBool, path, kafka_ips, topics):
+        try:
+
+            #kafka consumer
+            consumer = KafkaConsumer(
+                #bootstrap ip and port could probably be separate variables in the future
+                bootstrap_servers = kafka_ips,
+                auto_offset_reset='latest',
+                enable_auto_commit=False,
+                value_deserializer=lambda m: json.loads(m.decode('utf-8'))
+            )
+
+            #list of all topic names we want the consumer to subscribe to
+            subscribedTopics = []
+            foundTopics = False
+
+            #get all topic names
+            if not topics:
+                logger.log('INFO', f'No topics subscribed to')
```

Comment:
Should this result in a error exit? If there are no topics defined, they there's nothing to subscribe to and the component does nothing, right?